build/dsc/templates/dsc_type_spec.rb.erb

```diff
@@ -228,23 +228,22 @@ describe Puppet::Type.type(:dsc_<%= resource.friendlyname.downcase %>) do
 <%  end # if resource.ensurable? -%>
 <%  if resource.absentable? -%>
     describe "when dsc_ensure is 'absent'" do
-
       before(:each) do
-        dsc_<%= resource.friendlyname.downcase %>.original_parameters[:dsc_ensure] = 'absent'
-        dsc_<%= resource.friendlyname.downcase %>[:dsc_ensure] = 'absent'
+        dsc_<%= resource.friendlyname.downcase %>.original_parameters[:dsc_ensure] = '<%= resource.ensure_value %>'
```

Comment:
I believe you are correct. I did ask @jpogran to rebase #73 against this latest and ensure the specs pass. Which we can see they are not likely to. :)